src/object/cli_ec.c

```diff
@@ -1177,13 +1181,13 @@ obj_ec_recx_reasb(daos_iod_t *iod, d_sg_list_t *sgl,
 			tmp_recx.rx_nr = recx_end - full_end;
 			ec_data_recx_add(&tmp_recx, riod->iod_recxs, ridx,
 					 tgt_recx_idxs, oca, true);
-			ec_data_seg_add(&tmp_recx, iod_size, sgl, &iov_idx,
-					&iov_off, oca, iovs, iov_nr, sorter,
-					true);
+			ec_data_seg_add(&tmp_recx, iod_size, sgl,
+					&iov_idx, &iov_off, oca, iovs,
+					iov_nr, sorter, true);
```

Comment:
ditto, why you change the original indent,  not a big problem just that seems unnecessary and make the patch more difficult to review.

Reply:
Fixed.